qds_sdk/account.py

```diff
@@ -1,10 +1,64 @@
 """
 The Accounts module contains the base definition for a Qubole account object
 """
-
+import json
 from qds_sdk.resource import SingletonResource
+from qds_sdk.qubole import Qubole
+from qds_sdk.exception import ParseError
+from qds_sdk.util import GentleOptionParser
+from qds_sdk.util import OptionParsingError
+from qds_sdk.util import OptionParsingExit
+import argparse
+
+class AccountCmdLine:
+
+
+    @staticmethod
+    def parsers():
+        argparser = argparse.ArgumentParser(prog="qds.py account",
+                description="Account Creation for Qubole Data Service.")
+        subparsers = argparser.add_subparsers(title = "account operation")
+
+        #Create
+        create = subparsers.add_parser("create",
+                                       help="Create a new account")
+        create.add_argument("--name", dest="name", help="account name")
+        create.add_argument("--location", dest="defloc", help="Default location of S3")
+        create.add_argument("--access", dest = "acc_key", help="access key")
```

Comment:
Need better help message, for example: `AWS Access Key ID`, `AWS Secret Access Key`.